guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,577 @@
+
+{% include tip/close.html %}
+
+#### Auto-swapping to a default Cashu mint
+{% include image-gallery.html pages = page.images_auto-swap %}
+Wallets can provide the option to automatically swap ecash from an unknown mint to their default mint. This optional feature can simplify payments by routing all ecash through the assigned default mint. An auto-swap requires a lightning payment.
+
+{% include tip/open.html color="blue" icon="info" label="Auto-Swapping Involves Fees" %}
+
+It's important to inform users that auto-swapping involves fees and that the amount depends on the amount of ecash they want to swap and the current network conditions.
+
+{% include tip/close.html %}
+
+## Fedimint design guidelines
+
+### User interface and experience
+
+#### Federation status display
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/federation-guardian-info"
+   retina = "/assets/images/guide/how-it-works/ecash/federation-guardian-info@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/ecash/federation-guardian-info@2x.png"
+   alt-text = ""
+   width = 375
+   height = 812
+   layout = "float-right-desktop -background -shadow"
+   caption = "Example of a guardian status display in a wallet interface."
+%}
+
+Consider using a prominent, color-coded indicator to show the overall status of the federation. This should reflect whether the federation is fully operational, partially degraded, or offline. This helps users quickly assess the health of a federation.
+
+</div>
+#### Guardian Status Indicators
+{% include image-gallery.html pages = page.images_guardian-status %}
+
+Consider displaying real-time indicators of each guardian's status, such as the connection status and last activity. This information builds trust by keeping users informed about the reliability and performance of the guardians managing their funds.
+
+### Security and privacy
+
+#### Privacy considerations in guardian information display
+Fedimint automatically generates random names for guardians during setup. While users can change these names, it’s not recommended. When displaying guardian information, consider using code names like "Guardian 1" and randomly generated identicons for each guardian. This helps protect user and federation privacy, particularly for invite-only mints. Guardian information should only be visible to users who have joined the federation.
+
+### Metadata and information management
+
+#### Federation metadata fields: Explanation and usage
+
+Federations can provide configuration and metadata to users. These metadata fields are consensus-relevant, meaning they must be consistent across all federation members. This ensures that users can rely on their accuracy. Let's explore the core metadata fields defined in the Fedimint protocol:
```

Comment:
Can we link to the fedimint documentation here?

https://github.com/fedimint/fedimint/blob/master/docs/meta_fields/README.md